beebop/poppunkWrapper.py

```diff
@@ -37,6 +54,13 @@ def assign_clusters(self, dbFuncs, qc_dict, qNames):
         )
 
     def create_microreact(self, cluster_no):
+        """
+        Generates microreact visualisation output based on previous
+        assign-clusters() output.
+
+        Arguments:
+        cluster_no
```

Comment:
cluster_no needs a description (especially as I first read it as the opposite of cluster_yes, not as cluster_number :) )